debian/control

```diff
@@ -1,4 +1,4 @@
-Source: greenplum-db-oss
+Source: greenplum-db-5
```

Comment:
FYI: The Debian control file defines one source package (`Source`) and one or more binary packages (`Package`). The source package and binary package do not have to have the same name and often times they do not. The source name is used to identify the upstream source tarball while the package name is used to name the generated binary packages.

You could have

```config
Source: gpdb
...
Package: greenplum-db-5
```

That's not to say that we should do that here. I think changing them to say matched together makes sense, just wanted to share some additional context.